netmaster/docknet/docknet.go

```diff
@@ -249,29 +232,63 @@ func DeleteDockNet(tenantName, networkName, serviceName string) error {
 		log.Infof("Ignoring error in deleting docker network %s. Err: %v", docknetName, err)
 	}
 
+	err = DeleteDockNetState(tenantName, networkName, serviceName)
+	if docknetDeleted && err != nil {
```

Comment:
Is there a specific error here that we can check for rather than handling _all_ errors?